.eslintignore

```diff
@@ -0,0 +1 @@
+src/adapters/supabase/types/database.ts
```

Comment:
Do we want to commit the database file? I feel like it should get generated on build instead and never committed in the repo. Advantages would be that merging would be less likely to happen between contributors, and we would catch right away issues on build if the db schema has changed, as it would not compile.

Reply:
We always did it this way and I don't remember that we had any issues with this approach

Reply:
> Do we want to commit the database file? I feel like it should get generated on build instead and never committed in the repo. Advantages would be that merging would be less likely to happen between contributors, and we would catch right away issues on build if the db schema has changed, as it would not compile.

I'm partial towards this approach. We also have less control over the formatting/linting of generated files. If it's quick to generate we seem to have more to gain with FernandVEYRIER's approach. 

Another idea is to handle it on another pull?

Reply:
The downside I see is that external contributors would not be able to generate the file against the online DB, would be required to generate it against a local instance since they use the anon key. Still seems to make sense as this file wouldn't necessary reflect the state of the real db depending on the last time it got generated. I can make a task so we can think about it. Most likely not an emergency at this point for sure.

Reply:
Created an issue here https://github.com/ubiquibot/assistive-pricing/issues/2
I am good with the PR otherwise, after resolving the current comments!